scripts/setup/migrate.ps1

```diff
@@ -122,13 +122,38 @@ function Migrate {
     if ($destinationSvc.Status -eq [System.ServiceProcess.ServiceControllerStatus]::Running) {
         Stop-Service $destinationSvc.Name -ErrorAction Stop
     }
+    
+    $sslBindingInfo = $null
 
-    $userFiles = .\config.ps1 Get-UserFileMap
+    $oldServicePort = .\config.ps1 Get-ConfigPort -Path $Source
+    $newServicePort = .\config.ps1 Get-ConfigPort -Path $Destination
     
-    # Copy over Ssl bindings, in this case the source and destination are reversed
```

Comment:
The comment was to help explain the naming of `destinationPort` and `sourcePort`, but really the variables should have just been named better. Those variables have been changed to `oldServicePort` and `newServicePort`, and thus shouldn't require any additional info. Also, destination and source aren't reversed anymore since the variables aren't using those names.

The code now uses `oldServicePort` and `newServicePort` which is more clear. A migration goes from the

Reply:
The code is still a bit counter intuitive at a glance. We are copying newServicePort's binding info to $oldServicePort. And delete $newServicePort's binding.